content/2019/10/07/revert.rst

```diff
@@ -0,0 +1,31 @@
+public: yes
```

Comment:
Is this intentionally back-dated to Oct 7?

Reply:
yeah - we wouldn't have to -  but that's the date the video was posted. Probably not worth backdating like that, though. I'm just in the habit from pulling in long-past articles…